ui/app/src/services/users.ts

```diff
@@ -157,3 +158,29 @@ export function fetchMyRolesBySite(sites?: Site[]): Observable<LookupTable<strin
     )
   );
 }
+
+export function fetchGlobalPreferences(): Observable<LookupTable<any>> {
+  return get('/studio/api/2/users/me/properties').pipe(pluck('response', 'properties', ''));
+}
+
+export function fetchSitePreferences(siteId: string): Observable<LookupTable<any>> {
+  return get(`/studio/api/2/users/me/properties?siteId=${siteId}`).pipe(pluck('response', 'properties', siteId));
+}
+
+export function setPreferences(
+  properties: LookupTable<string>,
+  siteId?: string
+): Observable<{ response: ApiResponse; properties: LookupTable<any> }> {
+  return postJSON(siteId ? `/studio/api/2/users/me/properties?siteId=${siteId}` : '/studio/api/2/users/me/properties', {
```

Comment:
```suggestion
  return postJSON(`/studio/api/2/users/me/properties${siteId ? `?siteId=${siteId}` : ''}`, {
```

Reply:
I will remove the siteId, it is not needed as QS

Reply:
close